setup.py

```diff
@@ -154,7 +154,7 @@
     entry_points="""
     [console_scripts]
     wingproj=openmdao.devtools.wingproj:run_wing
-    webview=openmdao.devtools.webview:webview_argv
+    webview=openmdao.utils.webview:webview_argv
```

Comment:
Ah,  see that webview just moved (i assumed the new bokeh stuff was taking over for it).